enterprise/server/remote_execution/workspace/workspace.go

```diff
@@ -146,6 +149,43 @@ func (ws *Workspace) DownloadInputs(ctx context.Context, tree *repb.Tree) (*dirt
 	return txInfo, err
 }
 
+// AddCIRunner adds the BuildBuddy CI runner to the workspace root if it doesn't
+// already exist.
+func (ws *Workspace) AddCIRunner() error {
+	destPath := path.Join(ws.Path(), "buildbuddy_ci_runner")
+	exists, err := disk.FileExists(destPath)
+	if err != nil {
+		return err
+	}
+	if exists {
+		return nil
+	}
+	srcPath, err := bazelgo.Runfile("enterprise/server/cmd/ci_runner/ci_runner_/ci_runner")
```

Comment:
see my slack comment for more details, but this won't work on our deployed macs, the runner needs to be a part of the bundle

Reply:
Done -- updated `bundle.go` to fall back to runfiles, so that we can exclude the CI runner from the bundle during development.

The CI runner adds ~20MB to the bundle, bringing the total bundle size from 81MB => 101MB (with `-c opt`)